staff/dbenavente/SemanticMarks/leadtech-exercise/home.html

```diff
@@ -0,0 +1,234 @@
+                <ul>
+                    <li>
+                        <a href="https://leadtech.com/entrepreneur">Entrepreneurs</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/investor">Investors</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/contact">Contact</a>
+                    </li>
+                </ul>
+            </li>
+            <li>
+                <a href="https://leadtech.com/work-with-us">Work with us</a>
+                <ul>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/backend">Backend</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/bi">BI</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/content-and-copywriting">Content and copywriting</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/customer-service/">Customer Service</a>                        </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/finance">Finance</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/frontend">Frontend</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/human-resources">Human Resources</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/project-manager">Project Management</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/qa">QA</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/sem">SEM</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/seo">SEO</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/social-media">Social Media</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/sysadmin">Sysadmin</a>
+                    </li>
+                    <li>
+                        <a href="https://leadtech.com/work-with-us/ux">UX</a>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+        <h3>Headquarters</h3>
+        <p>Barcelona
+            Av. del Litoral 12-14, 5, Blue Building, 08005, Barcelona, Spain.
+            Tel: +34 934 98 96 20
+        </p>
+        <p>Badalona
+            Av. Maresme 44-46, 2a 26, 08918, Badalona, Spain
+        </p>
+        <p>United States
```

Comment:
This info could be wrapped in a `address` tag → https://developer.mozilla.org/es/docs/Web/HTML/Elemento/address